### ISSUE TITLE
LocalVariable.prototype.reassignPath run into a endless loop with rxjs.

### ISSUE THREAD

**Issue by RueDeRennes** [rollup/rollup#2210]
LocalVariable.prototype.reassignPath run into a endless loop with rxjs.

- rollup v. 0.59.2
- rxjs v. 6.1.0

that is a insight in my rollup.config.js
```js
const main = {
    input: index.ts,
    output: {
        banner: '',
        file: './dist/index.js',
        dir: './dist/',
        name: 'app',
        format: 'umd',
        sourcemap: true,
        exports: 'none',
        amd: { id: 'app' },
    })),
    context: 'this',
    external: [],
    plugins: [
        nodeResolve( {
                    include: path.join(_pkg.project.workspace, 'node_modules/**'),
                    extensions: ['.json', '.ts', '.html', '.css', '.scss'],
                    modulesOnly: false,
                    modules: true,
                    main: true,
                    jsnext: true,
                    preserveSymlinks: true,
                    customResolveOptions: {
                        preserveSymlinks: true
                    }
                }),
        commonjs({
                    include: path.join(_pkg.project.workspace, 'node_modules/**')
                }),
        typescript2({
                    useTsconfigDeclarationDir: true,
                    cacheRoot: _pkg.project.cache,
                    tsconfigOverride: {
                        compilerOptions: { declaration: _configureation.includes('debug') }
                    }
                }),
        sass(),
        sizes()
    ]
};
```

**Comment by lukastaegert** [comment]
Do you by chance have an example repo or could set one up? In order to fix the issue, we need to identify the exact code that triggers this. Otherwise it could take quite a while until we get to the bottom of this which would be a pity.

**Comment by lukastaegert** [comment]
Also, do you have a stack trace and a maximum call stack error or is it just "hanging"?

**Comment by RueDeRennes** [comment]
its tricky to set one up.
the anwnser on your second qestion is: 'it just hanging' (in `reassignPath` method loop).
when i removed rxjs or set it as external, it works fine.

**Comment by guybedford** [comment]
@RueDeRennes can you at least share your repo with the steps to reproduce this problem?

**Comment by guybedford** [comment]
(things like exact versions matter!)

**Comment by RueDeRennes** [comment]
it works! perfect!

**Comment by lukastaegert** [comment]
Fix released as 0.59.4
